source/features/landing-page/LandingPage.module.scss

```diff
@@ -62,7 +62,7 @@
 }
 
 .footer {
-  padding: 0 0 92px !important;
+  padding: 0 0 92px;
```

Comment:
why 92? instead of 32?

Reply:
Thats for larger resolutions. But for smaller screens we use now 32px.